src/comm.cpp

```diff
@@ -2317,6 +2317,14 @@ int close_socket(struct descriptor_data *d)
 //			GET_NAME(d->character));
 //		log(buf, 110, LOG_HMM);
 	}
+                        for(int iWear = 0; iWear < MAX_WEAR; iWear++)
+                             if(d->character->equipment[iWear])
+                                       obj_to_char( unequip_char( d->character, iWear,1 ), d->character );
+
+                        while(d->character->carrying)
+                            extract_obj(d->character->carrying);
+
+
```

Comment:
```suggestion
      for(int iWear = 0; iWear < MAX_WEAR; iWear++)
        if(d->character->equipment[iWear])
          obj_to_char( unequip_char( d->character, iWear,1 ), d->character );

        while(d->character->carrying)
          extract_obj(d->character->carrying);


```

Reply:
Nick, do you still think we need this after my last change in master from last night?

As for a spacing, I don't have a strong opinion. I think at the moment my Eclipse IDE settings (which are committed to this repo in .settings) are set to use tabs with 4 spaces per tab.

Reply:
Shouldn't be needed any more after yout changes no, @jhhudso.